api/relations/v1/check.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package api.relations.v1;
+import "google/api/annotations.proto";
+import "relations/v1/common.proto";
+option go_package = "ciam-rebac/api/relations/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.relations.v1";
+
+service CheckService {
```

Comment:
There are mixed conventions around the "Service" suffix. Buf suggests always including it as a matter of consistency to avoid one off cases that conflict with other names. Google Cloud goes the other way and suggests just having a one off when needed. I went with Buf style when there was a conflict.